components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -5842,6 +5900,7 @@ def __init__(self, dictionary, descriptorProvider):
                                          defaultValue=member.defaultValue,
                                          exceptionCode="return Err(());"))
             for member in dictionary.members]
+        assert not any(i.needsRooting for (_, i) in self.memberInfo)
```

Comment:
If RsaHashedKeyAlgorithm is the only code that would use this right now, I'm not bothered by this restriction.